telegram/chatmember.py

```diff
@@ -64,3 +64,11 @@ def de_json(data, bot):
         data['user'] = User.de_json(data.get('user'), bot)
 
         return ChatMember(**data)
+
+    def __eq__(self, other):
+        if isinstance(other, self.__class__):
+            return self.user == other.user and self.status == other.status
+        return NotImplemented
```

Comment:
I think this will be more appropriate:
`return type(self) == type(other) and self.user == other.user and self.status == other.status`
(no need to return `NotImplemented`